components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -243,6 +243,33 @@ sealed class LastAccessAction : BrowserAction() {
     ) : LastAccessAction()
 }
 
+/**
+ * [BrowserAction] implementations related to updating the [TabSessionState] inside [BrowserState]
+ * to track when media started playing in a page up until navigating away from that page.
+ */
+sealed class InProgressMediaAction : BrowserAction() {
+    /**
+     * Updates [TabSessionState.lastMediaAccess] for when media started playing in the tab identified by [tabId].
+     *
+     * @property tabId the ID of the tab to update.
+     * @property lastMediaAccess the value to signify when the tab last started playing media;
+     * defaults to [System.currentTimeMillis].
+     */
+    data class PlayStarted(
```

Comment:
nit: We currently have the convention of using a `Action` postfix for all actions which we found unneeded many time, but we should not mix and match. Let's keep this and simplify later if we want to.

Also, see comment above for a simpler way to model this actions?

Reply:
Will use just one UpdateLastMediaAccessAction. Thank you!